src/main/client/mrt.c

```diff
@@ -0,0 +1,74 @@
+#include <Python.h>
+
+#include <aerospike/aerospike_txn.h>
+#include "exceptions.h"
+#include "types.h"
+#include "client.h"
+
+PyObject *AerospikeClient_Commit(AerospikeClient *self, PyObject *args,
+                                 PyObject *kwds)
+{
+    AerospikeTransaction *py_transaction = NULL;
+
+    static char *kwlist[] = {"transaction", NULL};
+
+    if (PyArg_ParseTupleAndKeywords(args, kwds, "O!:commit", kwlist,
+                                    &AerospikeTransaction_Type,
+                                    (PyObject **)(&py_transaction)) == false) {
+        return NULL;
+    }
+
+    as_error err;
+    as_error_init(&err);
+
+    as_commit_status status;
+
+    Py_BEGIN_ALLOW_THREADS
+    aerospike_commit(self->as, &err, py_transaction->txn, &status);
+    Py_END_ALLOW_THREADS
+
+    if (err.code != AEROSPIKE_OK) {
+        raise_exception(&err);
+        return NULL;
+    }
+
+    PyObject *py_status = PyLong_FromUnsignedLong((unsigned long)status);
+    if (py_status == NULL) {
+        return NULL;
+    }
```

Comment:
nit: I don't think this check is necessary.

Reply:
I added this check as a habit of checking every C-API call to make sure it didn't return an error (this is a best practice)

Reply:
https://docs.python.org/3/c-api/exceptions.html